internal/services/network/availability_zone.go

```diff
@@ -0,0 +1,30 @@
+package network
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2020-06-01/resources"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+)
+
+func GetZones(ctx context.Context, client *resources.ProvidersClient, resourceType, location string) (*[]string, error) {
```

Comment:
But package loadbalancer will use this function too.

Reply:
then it should be duplicated for now and TODO's added as needed - this is intentionally private to discourage usage elsewhere (since this behaviour will be removed in 3.0)